cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -47,7 +47,8 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                 return new Editor({
                     parentElement: this.$('.modal-section'),
                     model: this.model,
-                    xblockType: this.options.xblockType
+                    xblockType: this.options.xblockType,
+                    enable_proctored_exams: this.options.enable_proctored_exams
```

Comment:
Can there be a more generic to make the options.enable_proctored_exams field available to the Editors? This place seems generic for all types of Editors.